docs/ops/signals/DFT_7.md

```diff
@@ -0,0 +1,209 @@
+## DFT <a name="DFT"></a> {#openvino_docs_ops_signals_DFT_7}
+
+**Versioned name**: *DFT-7*
+
+**Category**: Signal processing
+
+**Short description**: *DFT* layer performs the discrete complex-to-complex Fourier transformation of input tensor by specified dimensions.
+
+**Attributes**:
+
+    No attributes available.
+
+**Inputs**
+
+*   **1**: `data` - Input tensor of type *T* with data for the DFT transformation. Type of elements is any supported floating point type. The last dimension of the input tensor must be equal to 2, i.e. the input tensor shape must have the form `[D_0, D_1, ..., D_{N-1}, 2]`, representing the real and imaginary components of complex numbers in `[:, ..., :, 0]` and in `[:, ..., :, 1]` correspondingly.  Required.
+*   **2**: `axes` - 1D tensor of type *T_IND* specifying dimension indices where DFT is applied, and `axes` is any unordered list of indices of different dimensions of input tensor, e.g. `[0, 4]`, `[4, 0]`, `[4, 2, 1]`, `[1, 2, 3]`. These indices should be non-negative integers from `0` to `rank(data) - 1` inclusively.  Other dimensions do not change. The order of elements in `axes` attribute matters, and mapped directly to elements in the 3d input `signal_size`. Required.
```

Comment:
Let's add a comment that dft cannot be applied other the last dimension (which is equal to 2).

Reply:
And update the text that axes can be in the range  `[0, rank(data) - 2]`.

Reply:
Such comment have already existed:

> *   **Note**: The following constraint must be satisfied: `rank(data) >= len(axes) + 1 and input_shape[-1] == 2 and (rank(data) - 1) not in axes`.

Reply:
Fixed range of axes indices.